crates/rspack_plugin_javascript/src/visitors/swc_visitor/react.rs

```diff
@@ -81,40 +81,71 @@ module.hot.accept();
 RefreshRuntime.queueUpdate();
 "#;
 
-static HMR_FOOTER_AST: Lazy<Program> =
-  Lazy::new(|| parse_js_code(HMR_FOOTER.to_string(), &ModuleType::Js).expect("TODO:"));
+static HMR_FOOTER_AST: Lazy<Script> = Lazy::new(|| {
```

Comment:
Why change `Program` to `Script`?

Reply:
- Script body is Vec<Stmt> while Module body is Vec<ModuleItem> here, Vec<Stmt> can be transformed to Vec<ModuleItem> easily, we need to insert this ast body to both Module and Script